src/app/design/progressbar/progressbar.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'app-progressbar',
```

Comment:
```suggestion
  selector: 'design-progressbar',
```

Reply:
We decided to give base-components the prefix "design".

Reply:
Please don't, this only caused problems...

Reply:
Convinced Marcel. Design Prefix is fine, as long as there is an es-lint ignore annotation.

Reply:
That's bad, either you can tweak the eslint config or we don't do it.

Reply:
See https://github.com/cryptic-game/frontend/pull/346/commits/2a71c442c88f0ad45acdc2740ec3d20a28d84cc5.